src/App.java

```diff
@@ -1,47 +1,46 @@
 import java.util.ArrayList;
-
 import DLibX.DConsole;
 import net.java.games.input.Controller;
 import net.java.games.input.ControllerEnvironment;
 import utils.Gamepad;
+import menus.*;
 
 public class App {
+
+    // list vars
+    ArrayList<Menu> menus = new ArrayList<>();

```

Comment:
I was going to use this for something but decided agianst it. Will remove before merge.